### ISSUE TITLE
Link to client's content property detailed description doesn't work

### ISSUE THREAD

**Issue by ezz666** [awesomeWM/awesome#3473]
The link [https://awesomewm.org/apidoc/core_components/client.html#content](https://awesomewm.org/apidoc/core_components/client.html#content) led to the top of the Module:client page on all browsers I tried (firefox, chromium). Urls for all the other properties work fine.

**Comment by Elv13** [comment]
Indeed. That's because the DIV is `<div id='content'>` for the whole doc. That's pretty easy to fix (rename the CSS and the DIV to something else less prone to name collision), but I will wait until you submit your PR. Otherwise they will most probably conflict.
